docs/application/dotnet/guides/internationalization/localization.md

```diff
@@ -0,0 +1,253 @@
+            {
+                case "zh-CN":   // Chinese Simplified (People's Republic of China)
+                    netLanguage = "zh-Hans"; // correct code for .NET
+                    break;
+                case "zh-HK":  // Chinese Traditional (Hong Kong)
+                case "zh-hk":
+                case "zh-tw":  // Chinese Traditional (Taiwan)
+                case "zh-TW":
+                    netLanguage = "zh-Hant"; // correct code for .NET
+                    break;
+            }
+
+            Console.WriteLine("[Speedmeter] .NET Language/Locale:" + netLanguage);
+            return netLanguage;
+        }
+    }
+}
+
+```
+
+- Third. use the DependencyService in the Xamarin.Forms App to call the interface and set your `RESX` resources culture to the correct value.
+  you can receive culture info using `MessagingCenter` and update your app UI when your app is running and system Language has been changed.
+
+```cs
+var ci = DependencyService.Get<ILocalize>().CurrentCultureInfo;
+Resx.AppResources.Culture = ci; // set the RESX for resource localization
+DependencyService.Get<ILocalize>().SetLocale(ci);
+
+// Whenever language has been changed, CurrentCulture will be updated.
+MessagingCenter.Subscribe<ILocalize, CultureInfo>(this, "LanguageChanged", (obj, culture) =>
+{
+    Resx.AppResources.Culture = culture; 
+    DependencyService.Get<ILocalize>().SetLocale(culture);
+    //update your App UI
+});
+```
+
+
+<br><br>
+## Image Localization
+Tizen project supports localized images(resources) using different resource directories.
+Tizen uses res.xml file to specify information about the directory where localized resources(e.g. Image, Sound, and so on) are contained.
+The res.xml file is automatically generated when building your application.
+
+<br><br>
+**To add the localized resource directories in your application in Visual Studio:**
+
+- In Solution Explorer, Select Tizen project. And then, Go to Tools -> Tizen -> Resource Manager<br>
+![local_res_mgr_1](media/local_res_mgr_1.png)
+
+<br><br>
+- Resource Manager opens configurations -> Open the language drop-down list<br>
+![local_res_mgr_2](media/local_res_mgr_2.png)
+
+<br><br>
+- Select the language to add -> click Add button<br>
+![local_res_mgr_3](media/local_res_mgr_3.png)<br>
+![local_res_mgr_4](media/local_res_mgr_4.png)
+
+<br><br>
+- Resource directories are automatically created in Tizen project as follows.<br>
+  res.xml file will be generated in res directory of Tizen project after building an app.<br>
+![res.xml](media/local_res_xml.png)<br>
+![res.xml code](media/local_res_xml_code.png)
+
+<br><br>
+_Your application can sometimes run in a locale which you don't provide images for. In that case, Tizen loads the default image from the resource content directory (yourApp.Tizen/res/content/ ). If there is no default image within resource content directory and a device sets the locale which you don't provide images for, an error occurs._
```

Comment:
_Your application can sometimes run in a locale, for which you have not provided images. In that case, Tizen loads the default image from the resource content directory (yourApp.Tizen/res/content/ ). If there is no default image within the resource content directory and the device sets the locale, for which you have not provided images, an error occurs._

Can we give it as a Note?

Reply:
done